.cspell.json

```diff
@@ -4,6 +4,7 @@
     "fastfetch",
     "neofetch",
     "onefetch",
-    "uwufetch"
+    "uwufetch",
+    "pfetch"
```

Comment:
Nitpick: I'm trying to alphabetize these so they're easy to find as the list gets longer.